### ISSUE TITLE
MISC START ROW STAR COL NO WORK

### ISSUE THREAD

**Issue by roque-canales** [PlayUAV/PlayuavOSD#17]
value not affect at all of osd position on screen

**Comment by ghost** [comment]
When the value is positive it should work. If the value is negative, it will fail,  because the code defines the variable as unsigned int 16. Nevertheless, it is clear that this is a bug and need to be fixed.

**Comment by roque-canales** [comment]
Also I tested positive offset values, and noting change...

**Comment by N-Z-N** [comment]
Both the Row and Col values no longer work.
In v7 firmware (and v1.0.0.6 config tool), this is what I experienced:

"OSD display configtool: MISC ==> Start_Col setting.
 -Adding any negative value is not accepted.
     Eg. entering "-20" reverts to 65536 when value is refreshed (Read Params)
 -Positive values are ok. 
 -Also Start_Row works fine with negative values."

At that point Row field still worked with negative values.
Now it no longer does.
So both Col and Row values are not accepting negative values.

Additionally...making any changes to these fields in the POS side, does not reflect changes to the OSD either.
Basically it seems like both these values are completely non-functional as of the new release.

When I updated to v9 firmware (and v1.0.1.3 config tool),
I noticed that my entire OSD shifted down.
That is because I had the ROW set to (-10), when it was working on v7 firmware.
Now making any changes to ROW, has no affect.
